exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
```

Comment:
This prompting for a potential name of the method is not necessary, see: [circular-buffer](https://github.com/exercism/x-common/pull/488/files#diff-e447feb71d0e589a8d7bef71469adceb)
It has a comment, and then the list of cases.